qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -84,3 +84,6 @@
                     check_clock = 'cat /sys/devices/system/clocksource/clocksource0/available_clocksource | grep tsc'
                     cpu_model_flags += ",+invtsc"
                     flags = "nonstop_tsc"
+                - mce:
+                    cpu_model_flags += ",mce=off"
+                    flags = ""
```

Comment:
I guess you need add `no_flags = "mce"`.

Reply:
> I guess you need add `no_flags = "mce"`.

Thanks and update.
 (1/4) type_specific.io-github-autotest-qemu.x86_cpu_flags.others.mce: PASS (97.70 s)
 (2/4) type_specific.io-github-autotest-qemu.x86_cpu_flags.others.mce: PASS (71.75 s)
 (3/4) type_specific.io-github-autotest-qemu.x86_cpu_flags.others.mce: PASS (96.10 s)
 (4/4) type_specific.io-github-autotest-qemu.x86_cpu_flags.others.mce: PASS (155.41 s)